### PR TITLE
Seneca web integration

### PR DESCRIPTION
**Pull request by mirceaalexandru**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,8 +47,72 @@ This plugin module is included in the main Seneca module:
...
+
+   * `GET /my-api/zig`: `role:api,cmd:zig`
+   * `GET /my-api/bar`: `role:api,cmd:bar`
+   * `GET /my-api/qaz`: `role:api,cmd:qaz`
+   * `POST /my-api/qaz`: `role:api,cmd:qaz`
+
+```JavaScript
+
+var Chairo = require ( 'chairo' )
+const Hapi = require ( 'hapi' )
```
Comment (geek) [inline README.md]: var is fine here

**Comment by geek** [comment]
Looks good.  Will merge after chairo is updated and published

File: test/hapi.stat.test.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+var assert = require('assert')
```
Comment (AdrieanKhisbe) [inline test/hapi.stat.test.js]: Would it be better to use [code](https://github.com/hapijs/code) Instead?
(I can take care of this if you prefer)

File: README.md (hunk 1 above)
Comment (mirceaalexandru) [inline README.md]: @geek 

done

File: test/hapi.stat.test.js (hunk 2 above)
Comment (ckiss) [inline test/hapi.stat.test.js]: Of course @AdrieanKhisbe. You're free to replace `assert` with `Code` and issue a new PR. I think this PR is about to be merged, so I think it's best if you wait until then. (or you can do it now, it's up to you really :) thanks :+1:

Comment (AdrieanKhisbe) [inline test/hapi.stat.test.js]: Okey dokey.
I'll watch PR, and launch operation once PR landed :smiley:

**Comment by geek** [comment]
@mirceaalexandru this can't be merged at the moment, for the tests, feel free to point to your chairo branch for now instead of the npm version

File: .eslintrc [hunk 3]
```diff
@@ -1,3 +1,7 @@
 {
+  "rules": {
+    "space-in-parens": 0,
```
Comment (geek) [inline .eslintrc]: we should remove these overrides, and not have extra spaces in parens

**Pull request merged**